generation/scraper/baseRemap.rsp

```diff
@@ -11245,6 +11245,19 @@ __NCRYPT_PCP_HMAC_AUTH_SIGNATURE_INFO=NCRYPT_PCP_HMAC_AUTH_SIGNATURE_INFO
 __NCRYPT_PCP_TPM_FW_VERSION_INFO=NCRYPT_PCP_TPM_FW_VERSION_INFO
 __NCRYPT_PCP_RAW_POLICYDIGEST=NCRYPT_PCP_RAW_POLICYDIGEST
 _FN_OBJECT_MGR_FUNCTIONS=FN_OBJECT_MGR_FUNCTIONS
+_DEDUP_CHUNK_INFO_HASH32=DEDUP_CHUNK_INFO_HASH32
+_DedupDataPortManagerOption=DedupDataPortManagerOption
+_DedupDataPortVolumeStatus=DedupDataPortVolumeStatus
+_DedupDataPortRequestStatus=DedupDataPortRequestStatus
+_DedupChunkFlags=DedupChunkFlags
+_DedupHash=DedupHash
+_DedupChunk=DedupChunk
+_DedupStreamEntry=DedupStreamEntry
+_DedupStream=DedupStream
+_DedupChunkingAlgorithm=DedupChunkingAlgorithm
+_DedupHashingAlgorithm=DedupHashingAlgorithm
+_DedupCompressionAlgorithm=DedupCompressionAlgorithm
+_DEDUP_SET_PARAM_TYPE=DEDUP_SET_PARAM_TYPE
```

Comment:
@sotteson1 @mikebattista The initial build did not remap _DEDUP_SET_PARAM_TYPE. I do not have an opinion either way on whether this one should be re-mapped or not, but it seemed weird to have only one enum that started with an underscore, so I added it manually to the list. Is this the right thing to do?